### ISSUE TITLE
[Suggestion] Use UID/GID for permissions

### ISSUE THREAD

**Issue by jbonadiman** [IgnisDa/ryot#464]
Is it possible to instead of having to chown to 1001:1001, make the Dockerfile respect the UID/GID env variables? Are there any reasons not to?

**Comment by IgnisDa** [comment]
I don't think this is possible. Do you know of a project that does this?

**Comment by jbonadiman** [comment]
> I don't think this is even possible in docker. Dockerfiles do not have access to arbitrary env variables.
> 
> Do you know of a project that does this?

You're right, I thought this was done in the Dockerfile, but there's only calls to `useradd`/`groupadd` in it, which you already do.
I checked the repos of self-hosted apps that make use of this UID/PUID and GID/PGID env vars and what they do is set up a shell script to serve as the entrypoint, like here:
> https://github.com/thomiceli/opengist/blob/dcacde09595fafeddefa48bcd63c6cdd29e501ba/docker/entrypoint.sh
This allows any UID/GID set as env var to work, without having to chown the files in the host to match the user in the container.
But then, looking at it a bit more, I found out that changing the "user" in the service in the docker-compose, like this:
```yaml
ryot:
    user: 1000:1000
```
Also works, without the need to use chown.
Maybe put this as an alternative to chown in the host, in the documentation? 🤔
